PWGCF/FemtoUniverse/Tasks/femtoUniversePairTaskTrackTrackMultKtExtended.cxx

```diff
@@ -258,44 +258,78 @@ struct femtoUniversePairTaskTrackTrackMultKtExtended {
     // ConfNsigmaCombined -> TPC and TOF Sigma (combined) for momentum > ConfTOFpMin
 
     if (mom < twotracksconfigs.ConfTOFpMin) {
-      return TMath::Abs(nsigmaTPC) < twotracksconfigs.ConfNsigmaTPC;
+      if (TMath::Abs(nsigmaTPC) < twotracksconfigs.ConfNsigmaTPC) {
```

Comment:
lines 261-265 are equivalent to the old line 261. I don't see how that changes anything.
The only difference could be made by introducing line 274.

Reply:
Could you please keep the old code here except line 274?